resources/hpxml-measures/BuildResidentialHPXML/measure.rb

```diff
@@ -3493,7 +3493,7 @@ def self.set_air_infiltration_measurements(hpxml, runner, args)
 
   def self.set_roofs(hpxml, runner, model, args, sorted_surfaces)
     args[:geometry_roof_pitch] *= 12.0
-    if args[:geometry_attic_type] == HPXML::AtticTypeFlatRoof
+    if args[:geometry_attic_type] == HPXML::AtticTypeFlatRoof or args[:geometry_attic_type] == HPXML::AtticTypeBelowApartment
```

Comment:
Does this actually do anything? A MF unit that is below another unit shouldn't end up with a HPXML `Roof` (which is always adjacent to outside) but rather a HPXML `FrameFloor` with `ExteriorAdjacentTo="other housing unit"`.

Reply:
@shorowit `geometry_roof_pitch` can still affect the pitch of a PV system, which is why this fix was introduced.